.github/workflows/release-new-app.yml

```diff
@@ -0,0 +1,50 @@
+name: New release app
+
+on:
+   release:
+     types: [published]
+
+permissions:
+  contents: read
+  packages: read
+
+jobs:
+  create-archive:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Compress repo to Tar
+        uses: a7ul/tar-action@v1.1.0
+        id: compress
+        with:
+          command: c
+          cwd: ./
+          files: |
+            ./
+          outPath: pwa.tar.gz
+
+      - name: Upload Artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: zipped-artifact
+          path: ${{ github.workspace }}/pwa.tar.gz
+
+  send-archive:
+    runs-on: ubuntu-latest
+    needs: install-and-build-dependencies
+    steps:
+      - name: Download Artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: zipped-artifact
+          path: ./
```

Comment:
**suggestion**: Combine both jobs. Right now there's not much of a benefit to having the artefact.